ui/src/pages/case.triggers.js

```
@@ -0,0 +1,93 @@
+            try {
+                parsedData = JSON.parse(responseData);
+            } catch (e) {
+                console.error("Invalid JSON in response data:", e);
+                return; // Stop if the data is not valid JSON
+            }
+ 
+            const jsonCrackEmbed = document.getElementById("jsoncrackEmbed");
+ 
+            modal.show();
+
+            if (jsonCrackEmbed && jsonCrackEmbed.contentWindow) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)